### PR TITLE
Apply story padding via decorator using params

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue #3174

## Relevant technical choices

* Also removes `imagesloaded` script from preview head which is no longer used/needed.

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by danielgent** [comment]
@googlebot I consent.

File: stories/module-adsense-components.stories.js [hunk 1]
```diff
@@ -85,6 +85,7 @@ generateReportBasedWidgetStories( {
 			options: [],
 		},
 	},
+	padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-adsense-components.stories.js:88]: I know that it is requested in the IB, but please, do not set padding to zero for widgets because it makes it look worse. So, please, update all module widgets to have the default padding instead.

File: stories/module-adsense-settings.stories.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ storiesOf( 'AdSense Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-adsense-settings.stories.js:101]: Same with settings. We need padding for settings to make it look good in the storybook. Please, update all module stories to have the default padding for their settings stories.

File: stories/dashboard.stories.js [hunk 3]
```diff
@@ -68,6 +68,8 @@ storiesOf( 'Dashboard', module )
 				<PostSearcher />
 			</WithTestRegistry>
 		);
+	}, {
```
Comment (eugene-manuilov) [inline stories/dashboard.stories.js:71]: Dashboard stories also need to have the default padding.

File: stories/module-adsense.stories.js [hunk 4]
```diff
@@ -177,6 +179,8 @@ storiesOf( 'AdSense Module', module )
 				</Layout>
 			</WithTestRegistry>
 		);
+	}, {
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-adsense.stories.js:183]: We need the default padding for this story.

File: stories/module-adsense.stories.js [hunk 5]
```diff
@@ -247,5 +251,7 @@ storiesOf( 'AdSense Module', module )
...
```
Comment (eugene-manuilov) [inline stories/module-adsense.stories.js:255]: And for this one too. We need the default padding here.

File: stories/module-analytics.stories.js [hunk 6]
```diff
@@ -311,7 +311,10 @@ storiesOf( 'Analytics Module', module )
 	},
 	// This uses the legacy widget, the new one is in:
 	// 'Analytics Module/Components/Module Page/Acquisition Channels Widget'.
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
+	{
+		options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' },
+		padding: 0,
+	} )
```
Comment (eugene-manuilov) [inline stories/module-analytics.stories.js:317]: We do need padding here too. Please, update it to use the default one.

File: stories/module-analytics.stories.js [hunk 7]
```diff
@@ -358,4 +361,7 @@ storiesOf( 'Analytics Module', module )
 			</WithTestRegistry>
 		);
 	},
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } );
+	{
+		options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' },
+		padding: 0,
+	} );
```
Comment (eugene-manuilov) [inline stories/module-analytics.stories.js:367]: Same here, please, update it to use the default padding.

File: stories/module-search-console.stories.js [hunk 8]
```diff
@@ -88,4 +88,7 @@ storiesOf( 'Search Console Module', module )
 			</WithTestRegistry>
 		);
 	},
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } );
+	{
+		options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' },
+		padding: 0,
+	} );
```
Comment (eugene-manuilov) [inline stories/module-search-console.stories.js:94]: We need the default padding here too.

File: stories/settings.stories.js [hunk 9]
```diff
@@ -68,6 +68,7 @@ storiesOf( 'Settings', module )
 		options: {
 			delay: 3000, // Wait for tabs to animate.
 		},
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/settings.stories.js:71]: Please, use the default padding here.

File: stories/settings.stories.js [hunk 10]
```diff
@@ -95,6 +96,7 @@ storiesOf( 'Settings', module )
 		options: {
 			delay: 100, // Wait for screen to render.
 		},
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/settings.stories.js:99]: And here.

File: stories/settings.stories.js [hunk 11]
```diff
@@ -118,6 +120,8 @@ storiesOf( 'Settings', module )
 				</div>
 			</WithTestRegistry>
 		);
+	}, {
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/settings.stories.js:124]: And here.

File: stories/wp-dashboard.stories.js [hunk 12]
```diff
@@ -158,6 +158,7 @@ storiesOf( 'WordPress', module )
 			readySelector: '.googlesitekit-data-block',
 			delay: 2000, // Wait for table overlay to animate.
 		},
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/wp-dashboard.stories.js:161]: Please, use the default padding for all stories in this file.

File: stories/data-table.stories.js [hunk 13]
```diff
@@ -77,4 +77,5 @@ storiesOf( 'Global', module )
 			readySelector: '.googlesitekit-table-overflow',
 			delay: 2000, // Wait for table overflow to animate.
 		},
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/data-table.stories.js:80]: We need the default padding for this.

File: stories/modules-list.stories.js [hunk 14]
```diff
@@ -39,4 +39,6 @@ storiesOf( 'Global', module )
 				<ModulesList />
 			</WithTestRegistry>
 		);
+	}, {
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/modules-list.stories.js:43]: This story also needs the default padding.

File: stories/page-header.stories.js [hunk 15]
```diff
@@ -57,4 +57,6 @@ storiesOf( 'Global', module )
 				/>
 			</p>
 		</div>;
+	}, {
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/page-header.stories.js:61]: Same here, we need the default padding for this story.

File: stories/report-table.stories.js [hunk 16]
```diff
@@ -73,5 +73,6 @@ storiesOf( 'Global/ReportTable', module )
 				/>
 			</Layout>
 		);
-	} )
-;
+	}, {
+		padding: 0,
+	} );
```
Comment (eugene-manuilov) [inline stories/report-table.stories.js:78]: Same here. Please, use the default padding.

File: assets/js/components/surveys/CurrentSurvey.stories.js [hunk 17]
```diff
@@ -103,4 +103,5 @@ SurveyWithTermsStory.args = {
 
 export default {
 	title: 'Components/Surveys/CurrentSurvey',
+	parameters: { padding: 0 },
```
Comment (eugene-manuilov) [inline assets/js/components/surveys/CurrentSurvey.stories.js:106]: Please, use the default padding for these stories.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @danielgent. Technically speaking this is good to go, however, I think we need to revert some of the changes because they make many stories look worse. Left comments where we need to use the default padding instead of zero one. Please, take a look.

**Comment by danielgent** [comment]
Thanks for changing! All requested changes done I think

I'll fix the merge conflicts next and then good to go :+1:

File: stories/dashboard.stories.js [hunk 18]
```diff
@@ -135,7 +135,8 @@ storiesOf( 'Dashboard', module )
 			</WithTestRegistry>
 		);
 	},
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
+	{ padding: 0,
+	} )
```
Comment (eugene-manuilov) [inline stories/dashboard.stories.js:139]: @danielgent looks instead of removing the `padding` key, you removed the `options` key. Please, add the options back and remove the padding.

File: stories/module-analytics-components.stories.js [hunk 19]
```diff
@@ -145,6 +145,7 @@ generateAnalyticsWidgetStories( {
 		},
 	},
 	wrapWidget: false,
+	padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-analytics-components.stories.js:148]: Analytics widgets also need to have the default padding. Please, update all widgets in this file.

File: stories/module-analytics-settings.stories.js [hunk 20]
```diff
@@ -76,6 +76,7 @@ storiesOf( 'Analytics Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-analytics-settings.stories.js:79]: Same with the settings. We need to use the default padding for all settings stories in all modules.

File: stories/module-search-console-settings.stories.js [hunk 21]
```diff
@@ -58,6 +58,7 @@ const storyOptions = {
 			return <Story registry={ registry } />;
 		},
 	],
+	padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-search-console-settings.stories.js:61]: Same here, we need to use the default padding here.

File: stories/modules-tagmanager-settings.stories.js [hunk 22]
```diff
@@ -110,6 +110,7 @@ storiesOf( 'Tag Manager Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/modules-tagmanager-settings.stories.js:113]: And here. We need the default padding for all settings stories.

File: stories/wp-dashboard.stories.js [hunk 23]
```diff
@@ -198,6 +198,7 @@ storiesOf( 'WordPress', module )
...
```
Comment (eugene-manuilov) [inline stories/wp-dashboard.stories.js:201]: This is not needed.

File: stories/wp-dashboard.stories.js [hunk 24]
```diff
@@ -243,4 +244,5 @@ storiesOf( 'WordPress', module )
...
```
Comment (eugene-manuilov) [inline stories/wp-dashboard.stories.js:247]: And this one too.

File: stories/module-optimize-settings.stories.js [hunk 25]
```diff
@@ -76,6 +76,7 @@ storiesOf( 'Optimize Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-optimize-settings.stories.js:79]: All stories in this file also need to have the default padding.

File: stories/module-pagespeed-insights-components.stories.js [hunk 26]
```diff
@@ -63,6 +63,8 @@ storiesOf( 'PageSpeed Insights Module/Components', module )
 				<DashboardPageSpeedWidget { ...widgetComponentProps } />
 			</WithTestRegistry>
 		);
+	}, {
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-pagespeed-insights-components.stories.js:67]: All stories in this file also need the default padding.

File: stories/module-pagespeed-insights-settings.stories.js [hunk 27]
```diff
@@ -54,19 +54,22 @@ storiesOf( 'PageSpeed Insights Module/Settings', module )
 		decorators: [
 			withRegistry,
 		],
+		padding: 0,
```
Comment (eugene-manuilov) [inline stories/module-pagespeed-insights-settings.stories.js:57]: All stories in this file also need to have the default padding.

File: assets/js/components/surveys/SurveyQuestionRating.stories.js [hunk 28]
```diff
@@ -66,4 +66,5 @@ export default {
 			</div>
 		),
 	],
+	parameters: { padding: 0 },
```
Comment (eugene-manuilov) [inline assets/js/components/surveys/SurveyQuestionRating.stories.js:69]: We need to use the default padding for this story.

Comment (eugene-manuilov) [reply assets/js/components/surveys/SurveyQuestionRating.stories.js:69]: In addition to the default padding, we also need to fix this story and stories in the **`SurveyCompletion.stories.js`** file. We need to remove the custom decorator defined in the default export and move the wrapping `<div className="googlesitekit-survey"> to the template function.

File: assets/js/components/wp-dashboard/WPDashboardIdeaHub.stories.js [hunk 29]
```diff
@@ -67,4 +67,5 @@ export default {
 			return <Story />;
 		},
 	],
+	parameters: { padding: 0 },
```
Comment (eugene-manuilov) [inline assets/js/components/wp-dashboard/WPDashboardIdeaHub.stories.js:70]: We don't need this.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @danielgent. Added more findings. Please, take a look.

File: assets/js/components/surveys/SurveyQuestionRating.stories.js (hunk 28 above)
Comment (danielgent) [reply assets/js/components/surveys/SurveyQuestionRating.stories.js]: adds-wrapper
The custom decorator is needed so the stories don't look this

Comment (eugene-manuilov) [reply assets/js/components/surveys/SurveyQuestionRating.stories.js]: @danielgent oh... GH hid part of my comment. We need to move the wrapping div from the custom decorator into the template function that we use for stories. In this case, we will preserve the required markup and avoid using a custom decorator. Could you please try it on your end?

Comment (danielgent) [reply assets/js/components/surveys/SurveyQuestionRating.stories.js]: AHhh @eugene-manuilov  right now that makes sense :smile: 

Sure no problem will do now

Comment (danielgent) [reply assets/js/components/surveys/SurveyQuestionRating.stories.js]: Works because no VRT changes :+1: 

Back to you

File: assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.stories.js [hunk 30]
```diff
@@ -202,4 +202,5 @@ export default {
 			);
 		},
 	],
+	parameters: { padding: 0 },
```
Comment (eugene-manuilov) [inline assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.stories.js:205]: We don't need this too.
```suggestion
```

File: stories/dashboard.stories.js [hunk 31]
```diff
@@ -135,7 +135,11 @@ storiesOf( 'Dashboard', module )
 			</WithTestRegistry>
 		);
 	},
-	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
+	{
+		options: {
+			readySelector: '.googlesitekit-chart .googlesitekit-chart__inner',
+		},
+	} )
```
Comment (eugene-manuilov) [inline stories/dashboard.stories.js:142]: Let's revert these changes too to exclude this file from this PR
```suggestion
	{ options: { readySelector: '.googlesitekit-chart .googlesitekit-chart__inner' } } )
```

File: stories/module-analytics-settings.stories.js [hunk 32]
```diff
@@ -235,10 +235,28 @@ storiesOf( 'Analytics Module/Settings', module )
...
+		usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: false, gaPermission: true } ),
+	)
+	.add(
+		'Existing Tag w/o access, GTM property w/ access',
+		usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: true, gaPermission: false } ),
+	)
+	.add(
+		'Existing Tag w/o access, GTM property w/o access',
+		usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: false, gaPermission: false } ),
+	)
```
Comment (eugene-manuilov) [inline stories/module-analytics-settings.stories.js:261]: This can also be reverted to exclude this file from the PR
```suggestion
	.add( 'No Tag, GTM property w/ access', usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: false, gtmPermission: true } ) )
	.add( 'No Tag, GTM property w/o access', usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: false, gtmPermission: false } ) )
	.add( 'Existing Tag w/ access, GTM property w/ access', usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: true, gaPermission: true } ) )
	.add( 'Existing Tag w/ access, GTM property w/o access', usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: false, gaPermission: true } ) )
	.add( 'Existing Tag w/o access, GTM property w/ access', usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: true, gaPermission: false } ) )
	.add( 'Existing Tag w/o access, GTM property w/o access', usingGenerateGTMAnalyticsPropertyStory( { useExistingTag: true, gtmPermission: false, gaPermission: false } ) )
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Ok, this is almost good to go. Added a few suggestions that I'll apply and approve this PR. Thank you very much, @danielgent!

**Review by eugene-manuilov — APPROVED** [review]
